cmd/skaffold/app/cmd/flags.go

```diff
@@ -509,6 +509,14 @@ var flagRegistry = []Flag{
 		DefinedOn:     []string{"debug", "dev", "run"},
 		IsEnum:        true,
 	},
+	{
+		Name:          "build-concurrency",
+		Usage:         "Number of concurrently running builds",
```

Comment:
Can you be a bit more verbose:
 `Number of concurrently running builds. Set to 0 to run all builds in parallel. Doesn't violate build order among dependencies.`

Reply:
Sorry for the bits-n-pieces review.

Reply:
will have to run `./hack/generate-man.sh` script again to update the docs